### ISSUE TITLE
Clickable links

### ISSUE THREAD

**Issue by DonaldTsang** [DangerOnTheRanger/maniwani#82]
The Markdown is not complete, it needs clickable links, and possibly an "opt-out" option like 8/pol/

**Comment by DangerOnTheRanger** [comment]
Doing this properly (making sure only URLs with proper domains get highlighted instead of just regexing `.+\..+`, making sure HTTPS is handled where linked, etc.) could take a little bit of effort, but I think many people would like this. I'll accept a PR for this if someone beats me to implementing it.

**Comment by DonaldTsang** [comment]
> making sure only URLs with proper domains get highlighted
What if we just use a standard URL regex?

**Comment by DangerOnTheRanger** [comment]
True, that would simplify things a fair amount. It's not a high priority of mine, but again, I'd accept a PR for this.
